lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -363,22 +364,34 @@ void LayerTreeItem::saveAnonymousLayer()
 
     std::string fileName;
     if (sessionState->saveLayerUI(nullptr, &fileName)) {
+
+        std::string        filePath(fileName);
```

Comment:
Not sure why we need to create a new variable here. I think we could process fileName directly.

Reply:
Removed the extra variable.